File: dayTwo/Python/levelTwo/task_five.py
first_number = int(input("Enter first number: "))
second_number = int(input("Enter second number: "))
third_number = int(input("Enter third number: "))

largest_number = 0;
        
if(first_number > second_number and first_number > third_number):
    largest_number = first_number
elif(second_number > first_number and second_number > third_number):
    largest_number = second_number
else:
    largest_number = third_number     
    
print(largest_number)   

File: dayTwo/Java/levelTwo/TaskFive.java
import java.util.Scanner;
public class TaskFive{

    public static void main(String[] args){
    
        Scanner inputCollector = new Scanner(System.in);
        System.out.print("Enter first number: ");
        int firstNumber = inputCollector.nextInt();
        
        System.out.print("Enter second number: ");
        int secondNumber = inputCollector.nextInt();
        
        System.out.print("Enter third number: ");
        int thirdNumber = inputCollector.nextInt();
      
        int largestNumber = 0;
        
        if(firstNumber > secondNumber && firstNumber > thirdNumber)
            largestNumber = firstNumber;
        else if(secondNumber > firstNumber && secondNumber > thirdNumber)
            largestNumber = secondNumber;
        else 
            largestNumber = thirdNumber;
            
        System.out.print(largestNumber);  
    
    }

}
